test/migrate.js

```diff
@@ -0,0 +1,15 @@
+const 
+	knexFile = require('../knexfile.js')
+	knex = require('knex')(knexFile)
+	knexMigrate = require('knex-migrate')
+	mv = '20180507173236'
+
+async function run() {
+	const log = ({ action, migration }) => console.log('Doing ' + action + ' on ' + migration)
```

Comment:
Prefer template strings

Reply:
sure can do, let me look for a better syntax highlighting solution, its hard to read on longer strings without the concats showing where the vars are

Reply:
If you're not using VSCode you're doing it wrong. 😛